### ISSUE TITLE
Prevent invalid locale messages to throw errors (leading to 500s)

### ISSUE THREAD

**Issue by willdurand** [mozilla/addons-frontend#9107]
In https://github.com/mozilla/addons-frontend/issues/9105, the issue was an invalid locale message that contained the same placeholder twice:

```
"într-una dintre %(extensionStart)sextensiile%(extensionEnd)s%(extensionEnd)s " 
```

This somehow generated an error and the app served a 500 error page instead of the expected (404) page.

We should add a test case or some other check to ensure this won't happen again.

**Comment by devil-cyber** [comment]
can I work on this I am new to Mozilla

**Comment by willdurand** [comment]
No, AFAIK but that's not an issue suitable for contributors. Sorry.

**Comment by KevinMind** [comment]
Old Jira Ticket: https://mozilla-hub.atlassian.net/browse/ADDFRNT-172
